server/socketCommands/socketFunctions/game.ts

```diff
@@ -37,12 +43,29 @@ export const gameFunctions = (socket: Socket, io: Server) =>{
     game.round = nextRound
     game.guessOrder = []
     if(game.gameClock) clearInterval(game.gameClock)
-    console.log(round)
     io.to(roomName).emit('update_drawer_and_round', { drawerIdx: nextDrawerIdx, round: nextRound, players })
   })
 
+  socket.on("end_of_round", (roomName)=>{
+    io.to(roomName).emit("end_of_round_to_client")
+  })
+  
+  socket.on("get_roundend_scoreboard", (roomName)=>{
+    const game = games[roomName]
+    const { players } = game
+    io.to(roomName).emit('send_scoreboard_to_client', players)
+    
+    const scoreBoardTimer = setInterval((elapsedSeconds = 0) => {
```

Comment:
Probably not an issue right now, but I wonder if this has a chance of starting multiple timers and cause the same bug that we had before. It might not cause the error in this specific case, but it might as we add stuff in the future. It might make more sense to put the timer on ```game``` and then only start a timer if there is no timer yet.